uint8 GET_RELATION_MATRICES=0
uint8 IS_POTENTIAL_MATCH=1
uint8 INFER_RELATION_MATRICES=2
uint8 GET_ALL_INSTANCES=3
uint8 PROLOG_QUERY=4
uint8 GET_INSTANCE_COUNT=5
uint8 header
string data
---
uint8 GET_RELATION_MATRICES=0
uint8 IS_POTENTIAL_MATCH=1
uint8 INFER_RELATION_MATRICES=2
uint8 GET_ALL_INSTANCES=3
uint8 PROLOG_QUERY=4
uint8 GET_INSTANCE_COUNT=5
uint8 FAILED=100
uint8 header
string data
